1:52:08 PM: src/Section/Footer.tsx(28,90): error TS2339: Property 'block' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Button> & Readonly<{ children?: ReactNode; }> & Readonly<ContainerProps>'.

CI 깨지는걸 수정해줘

--- a/docs/components/Button.mdx
+++ b/docs/components/Button.mdx
@@ -47,15 +47,15 @@ import { Component as Button } from './Button';
 <Playground>
   <Button leftIconSrc="https://s3.ap-northeast-2.amazonaws.com/class101-ui/images/ic-plus-black.png">Click me</Button>
   <Button rightIconSrc="https://s3.ap-northeast-2.amazonaws.com/class101-ui/images/ic-plus-black.png">Click me</Button>
-  <Button leftIcon={<Icon.Add fillColor={Colors.white} size={16} />}>Click me</Button>
-  <Button rightIcon={<Icon.Add fillColor={Colors.white} size={16} />}>Click me</Button>
+  <Button leftIcon={<Icon.Add fillColor={Colors.white} />}>Click me</Button>
+  <Button rightIcon={<Icon.Add fillColor={Colors.white} />}>Click me</Button>
   <Button>
     <Icon.Add fillColor={Colors.white} size={16} />
-    Click me
+    <span style={{ marginLeft: '4px' }}>Click me</span>
   </Button>
   <Button>
+    <span style={{ marginRight: '4px' }}>Click me</span>
     <Icon.Add fillColor={Colors.white} size={16} />
-    Click me
   </Button>
 </Playground>
 

--- a/docs/formControls/ControlGroup.mdx
+++ b/docs/formControls/ControlGroup.mdx
@@ -34,12 +34,12 @@ Input을 Label로 묶어줍니다.
 ## Block control group
 
 <Playground>
-  <ControlGroup block>
+  <ControlGroup fill>
     <Input placeholder="Placeholder text" />
     <Button size="lg">Click me</Button>
   </ControlGroup>
   <br />
-  <ControlGroup block>
+  <ControlGroup fill>
     <Input placeholder="Placeholder text" size="sm" />
     <Button size="md">Click me</Button>
   </ControlGroup>

--- a/src/Button/index.tsx
+++ b/src/Button/index.tsx
@@ -82,14 +82,24 @@ const StyledSpinner = styled(Spinner)<StyledSpinnerProps>`
 `;
 
 const LeftIcon = styled.div`
-  img {
+  > img {
     width: 100%;
+    height: 100%;
+  }
+  > svg {
+    width: 100%;
+    height: 100%;
   }
 `;
 
 const RightIcon = styled.div`
-  img {
+  > img {
     width: 100%;
+    height: 100%;
+  }
+  > svg {
+    width: 100%;
+    height: 100%;
   }
 `;
 

--- a/src/Section/Footer.tsx
+++ b/src/Section/Footer.tsx
@@ -1,5 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
+
 import Button from '../Button';
 import { gray200, gray600, white } from '../Colors';
 
@@ -25,7 +26,7 @@ export default ({
   ...restProps
 }: Props) => (
   <Footer>
-    <Button color={color} backgroundColor={backgroundColor} style={style} {...restProps} block>
+    <Button color={color} backgroundColor={backgroundColor} style={style} {...restProps} fill>
       {buttonTitle}
     </Button>
   </Footer>

